content/events/2024/12-10-podman.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2024-12-10T19:00:00
+publishDate: 2024-11-29
+#register: "https://www.helloasso.com/associations/bjpc/evenements/decembre-2024"
+tags:
+- tools
+- Docker
```

Comment:
```suggestion
- kubernetes
- tools
```
Minor update to stick with the existing tags to keep the tag list cardinality low